core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -2527,7 +2529,7 @@ private void setAttributeFromEntry(SetAttributeEntry entry)
     }
     try (LockedInodePath inodePath = mInodeTree
         .lockFullInodePath(entry.getId(), InodeTree.LockMode.WRITE)) {
-      setAttributeInternal(inodePath, entry.getOpTimeMs(), options);
+      setAttributeInternal(inodePath, false, entry.getOpTimeMs(), options);
```

Comment:
Typically, the replay flag should be true for replaying journal entries, and the client requests should set `replayed` to false.